wheel/src/api.rs

```diff
@@ -146,8 +149,10 @@ pub fn get_puzzle_and_solution_for_coin<'a>(
     let program = py_to_slice::<'a>(program);
     let args = py_to_slice::<'a>(args);
 
-    let program = node_from_bytes_backrefs(&mut allocator, program)?;
-    let args = node_from_bytes_backrefs(&mut allocator, args)?;
```

Comment:
these used to return `std::io::Result<>`, and that's why they could implicitly be converted to `PyErr`, is that right?

Reply:
yep